api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -791,4 +791,41 @@ public void getCountOfDrugs_shouldReturnTheTotalNumberOfMatchingNumbers() {
 		assertEquals(new Integer(1),
 		    conceptService.getCountOfDrugs(phrase, conceptService.getConcept(conceptId), true, true, true));
 	}
+
+	/**
+	 * @see ConceptServiceImpl#saveConceptProposal(ConceptProposal) 
+	 */
+	@Test
+	public void saveConceptProposal_shouldReturnSavedConceptProposalObject() {
+		String ORIGINAL_TEXT = "OriginalText";
```

Comment:
Is this a constant? Or is it allowed to change its value?

Reply:
I'm really, really sorry for overlooking so many things! Thank you for the reminders, I will do my best to make sure it doesn't happen again.

Reply:
Please do not worry at all. It happens to all of us. 😊